.vscode/c_cpp_properties.json

```diff
@@ -1,18 +1,25 @@
 {
-  "configurations": [
-    {
-      "name": "Pico",
-      "includePath": [
-        "${workspaceFolder}/**",
-        "${env:PICO_SDK_PATH}/**"
-      ],
-      "defines": [],
-      "compilerPath": "${env:PICO_INSTALL_PATH}/gcc-arm-none-eabi/bin/arm-none-eabi-gcc.exe",
-      "cStandard": "c11",
-      "cppStandard": "c++11",
-      "intelliSenseMode": "linux-gcc-arm",
-      "configurationProvider": "ms-vscode.cmake-tools"
-    }
-  ],
-  "version": 4
+	"configurations": [
```

Comment:
Question: Will users be able to choose which configuration to use or will VSCode automatically detect the proper one to use?

Reply:
In the command palette, there is an option for **C/C++: Select a Configuration...**.